### PR TITLE
chore: add Susmita-Dey

### PR DESCRIPTION
**Pull request by Susmita-Dey**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1009"></a>

### CODE REVIEW

**Review by syedsohan — CHANGES_REQUESTED** [review]
Could you please revert (un-commit, not delete) `README.md` and `package-lock.json` files?

**Comment by Susmita-Dey** [comment]
> Could you please revert (un-commit, not delete) `README.md` and `package-lock.json` files?

How to do that?

**Comment by syedsohan** [comment]
> > Could you please revert (un-commit, not delete) `README.md` and `package-lock.json` files?
> 
> How to do that?

- Copy the current `package-lock.json` from the `main branch` into `your branch` and then `replace and commit the changes.`
- Copy the current `README.md` from the `main branch` into `your branch` and then `replace and commit the changes.`

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Your profile looks good, just need to remove the readme and package-lock files from this pull request. If you want the hackerrank icon and it doesn't exist feel free to raise an issue for it to get it added :)

**Comment by Susmita-Dey** [comment]
> > > Could you please revert (un-commit, not delete) `README.md` and `package-lock.json` files?
> > 
> > 
> > How to do that?
> 
> * Copy the current `package-lock.json` from the `main branch` into `your branch` and then `replace and commit the changes.`
> * Copy the current `README.md` from the `main branch` into `your branch` and then `replace and commit the changes.`

How do I replace the changes? Shall I create a new pull request for that?

**Comment by emmalearnscode** [comment]
> How do I replace the changes? Shall I create a new pull request for that? I have made the changes in the README.md file and package-lock.json in my forked repository.

No need to create a new pull request. The readme file is now gone from this pull request so that worked. There is just a problem with the versions in the package-lock file not being the same which is why it's still there. I will make a comment telling you which line to change.

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.59.1",
+      "version": "0.60.2",
```
Comment (emmalearnscode) [inline package-lock.json:9]:
```suggestion
      "version": "0.59.1",
```

Comment (Susmita-Dey) [reply package-lock.json:9]: I have commited the changes from here. Hope that it has been resolved.

**Comment by Susmita-Dey** [comment]
> Please remove the files that belong to the hackerrank icon from the PR, create a different branch in your project and make a pull request from that branch instead.

Okay, I'm creating a pull request for that.

**Comment by emmalearnscode** [comment]
@Susmita-Dey Your hackerrack pull request has been merged so you should be able to fix this PR now :)

**Comment by Susmita-Dey** [comment]
> @Susmita-Dey Your hackerrack pull request has been merged so you should be able to fix this PR now :)

Okay, I'm trying to do so, but I'm unable to fetch upstream and the branch is 23 commits behind. 
I'm facing this issues but still unable to fetch upstream. 
image
image
Please help me

**Comment by emmalearnscode** [comment]
Either you can do as it says and click on open pull request where you will need to go through and follow the instructions to fix the merge conflicts (which is good practice in how to do it) or you can delete and refork the repo so you are working with the most updated version. One way to avoid this problem in the future is to make your changes in a branch, that way you shouldn’t end up with conflicts when fetching upstream.

**Pull request closed without merging**